orca_detector/vggish_model.py

```diff
@@ -212,9 +213,9 @@ class OrcaVGGish(VGGish):
 
     def custom_preprocessing_layers(self):
         """
-            TODO: implement BatchNormalization of inputs
+            Applying batch normalization.
         """
-        pass
+        self.x = BatchNormalization()(self.audio_input)
```

Comment:
I assume the BN improved the results, right?

Reply:
I didn't do an immediate before/after test, but it just seemed good hygiene to add this in since there was a lot of variation in how samples were collected.